src/app/Plans/components/PlanDetails.tsx

```diff
@@ -0,0 +1,183 @@
+import MappingStatus from '@app/Mappings/components/MappingStatus';
+import { warmCriticalConcerns, someVMHasConcern } from './Wizard/helpers';
+interface IPlanDetailsProps {
+  plan: IPlan;
+}
+
+const PlanDetails: React.FunctionComponent<IPlanDetailsProps> = ({ plan }: IPlanDetailsProps) => {
+  usePausedPollingEffect();
+
+  const networkMappings = useMappingsQuery(MappingType.Network);
+  const networkMapping =
+    networkMappings.data?.items.find(
+      (mapping) => (mapping.metadata as IMetaObjectMeta).name === plan.spec.map.network.name
+    ) || null;
+
+  const storageMappings = useMappingsQuery(MappingType.Storage);
+  const storageMapping =
+    storageMappings.data?.items.find(
+      (mapping) => (mapping.metadata as IMetaObjectMeta).name === plan.spec.map.storage.name
+    ) || null;
+
+  const providers = useInventoryProvidersQuery();
+  const provider =
+    providers.data?.vsphere.find((provider) => provider.name === plan.spec.provider.source.name) ||
+    null;
+
+  const vms = useVMwareVMsQuery(provider);
+  const selectedVMs =
+    vms.data?.filter((vm) => plan.spec.vms.find((planVM) => planVM.id === vm.id)) || [];
+
+  const warmCriticalConcernsFound = plan.spec.warm
+    ? warmCriticalConcerns.filter((label) => someVMHasConcern(selectedVMs as IVMwareVM[], label))
+    : [];
+
+  return (
+    <Grid hasGutter className={`${spacing.mtSm} ${spacing.mbMd}`}>
+      <GridItem md={12}></GridItem>
+      <GridItem md={3}>Plan name</GridItem>
+      <GridItem md={9} id="review-plan-name">
+        {plan.metadata.name}
+      </GridItem>
+      {plan.spec.description ? (
+        <>
+          <GridItem md={3} id="plan-description-label">
+            Plan description
+          </GridItem>
+          <GridItem md={9} id="review-plan-description" aria-labelledby="plan-description-label">
+            {plan.spec.description}
+          </GridItem>
+        </>
+      ) : null}
+      <GridItem md={3} id="source-provider-label">
+        Source provider
+      </GridItem>
+      <GridItem md={9} id="review-source-provider" aria-labelledby="source-provider-label">
+        {plan.spec.provider.source.name}
+      </GridItem>
+      <GridItem md={3} id="target-provider-label">
+        Target provider
+      </GridItem>
+      <GridItem md={9} id="review-target-provider" aria-labelledby="target-provider-label">
+        {plan.spec.provider.destination.name}
+      </GridItem>
+      <GridItem md={3} id="target-namespace-label">
+        Target namespace
+      </GridItem>
+      <GridItem md={9}>
+        <span id="review-target-namespace" aria-labelledby="target-namespace-label">
+          {plan.spec.targetNamespace}
+        </span>
+      </GridItem>
+      <GridItem md={3} id="transfer-network-label">
+        Migration transfer network
+      </GridItem>
+      <GridItem md={9} id="review-transfer-network" aria-labelledby="transfer-network-label">
+        {plan.spec.transferNetwork || POD_NETWORK.name}
```

Comment:
I found a bug here. If you open the details for a plan that has a transfer network other than the pod network, it crashes:

```
Uncaught Error: Objects are not valid as a React child (found: object with keys {name, namespace}). If you meant to render a collection of children, use an array instead.
    at throwOnInvalidObjectType (react-dom.development.js?7f13:13232)
    at reconcileChildFibers (react-dom.development.js?7f13:14134)
    at reconcileChildren (react-dom.development.js?7f13:16991)
    at updateHostComponent (react-dom.development.js?7f13:17633)
    at beginWork (react-dom.development.js?7f13:19081)
    at HTMLUnknownElement.callCallback (react-dom.development.js?7f13:3946)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?7f13:3995)
    at invokeGuardedCallback (react-dom.development.js?7f13:4057)
    at beginWork$1 (react-dom.development.js?7f13:23965)
    at performUnitOfWork (react-dom.development.js?7f13:22777)
throwOnInvalidObjectType @ react-dom.development.js?7f13:13232
reconcileChildFibers @ react-dom.development.js?7f13:14134
reconcileChildren @ react-dom.development.js?7f13:16991
updateHostComponent @ react-dom.development.js?7f13:17633
beginWork @ react-dom.development.js?7f13:19081
callCallback @ react-dom.development.js?7f13:3946
invokeGuardedCallbackDev @ react-dom.development.js?7f13:3995
invokeGuardedCallback @ react-dom.development.js?7f13:4057
beginWork$1 @ react-dom.development.js?7f13:23965
performUnitOfWork @ react-dom.development.js?7f13:22777
workLoopSync @ react-dom.development.js?7f13:22708
renderRootSync @ react-dom.development.js?7f13:22671
performSyncWorkOnRoot @ react-dom.development.js?7f13:22294
eval @ react-dom.development.js?7f13:11328
unstable_runWithPriority @ scheduler.development.js?b589:468
runWithPriority$1 @ react-dom.development.js?7f13:11277
flushSyncCallbackQueueImpl @ react-dom.development.js?7f13:11323
flushSyncCallbackQueue @ react-dom.development.js?7f13:11310
discreteUpdates$1 @ react-dom.development.js?7f13:22421
discreteUpdates @ react-dom.development.js?7f13:3757
dispatchDiscreteEvent @ react-dom.development.js?7f13:5890
react-dom.development.js?7f13:13232 Uncaught Error: Objects are not valid as a React child (found: object with keys {name, namespace}). If you meant to render a collection of children, use an array instead.
    at throwOnInvalidObjectType (react-dom.development.js?7f13:13232)
    at createChild (react-dom.development.js?7f13:13470)
    at reconcileChildrenArray (react-dom.development.js?7f13:13720)
    at reconcileChildFibers (react-dom.development.js?7f13:14126)
    at reconcileChildren (react-dom.development.js?7f13:16991)
    at updateFragment (react-dom.development.js?7f13:17295)
    at beginWork (react-dom.development.js?7f13:19103)
    at HTMLUnknownElement.callCallback (react-dom.development.js?7f13:3946)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?7f13:3995)
    at invokeGuardedCallback (react-dom.development.js?7f13:4057)
```

This is because `plan.spec.transferNetwork` is an object, not a string. You need `plan.spec.transferNetwork.name`.

Reply:
Done